de/_posts/2019-12-07-rails-beginner-workshop.md

```diff
@@ -0,0 +1,55 @@
+---
+tags: beginner rails_beginner_workshop
+categories: de events
+slug: Anfänger_innen-Workshop
+i18n_key: rails_beginner_workshop_2019_12
+---
+
+![Foto von Dutzenden Frauen mit Laptops, die zusammen um einen großen weißen Tisch herum programieren.](/assets/images/Dec19-Banner.jpeg)
```

Comment:
Dutzenden => dutzenden, it's not part of the noun but a counting-adjective as far as I am concerned.

prorgramieren => programmieren